shared/src/helpers/step-helps.cjsx

```diff
@@ -1,5 +1,6 @@
 React = require 'react'
 _ = require 'underscore'
+capitalize = require 'lodash/capitalize'
 
```

Comment:
I like requiring individual stuff this way.  I think it'd be a good thing to do when/if we drop underscore